contents/tutorials/hogql-date-time-filters.md

```diff
@@ -0,0 +1,120 @@
+
+![hogql.mp4](../images/tutorials/hogql-date-time-filters/hogql.mp4)
+
+## Accessing your data’s dates and times
+
+To filter on dates and times, you need access to those values. You can find a full list of events, properties, and types in your [data management tab](https://app.posthog.com/data-management/events), but here is a summary of the important points:
+
+- You can access the time events happened with the `timestamp` property which is automatically set on events.
+- Persons have a `created_at` property which you can access with `person.created_at`.
+- For custom event or person properties, such as `subscribed_at`, you can access them with `person.properties.subscribed_at`.
+
+> **Note:** you might need to convert strings to date time objects with the `toDateTime()` function or convert date time objects to strings with `toString()`.
+
+The [ClickHouse SQL statements](https://clickhouse.com/docs/en/sql-reference) we built HogQL on also have useful helper functions to know when it comes to using dates. These include:
+
+- `now()`: the current date and time at the moment of query analysis.
+- `today()`: the current date at the moment of query analysis.
+- `yesterday()`: yesterday’s date at the moment of query analysis.
+
+## Events from a specific time range
+
+To filter for events in a specific time range, you can use `toDateTime()` to create a date time object that we compare with the event `timestamp`. 
+
+For example, if we wanted to filter for events after 9:26 AM on September 16th, 2022, and before 2:34 PM on October 1st, 2022, we can use the expression:
+
+```
+toDateTime('2022-09-16 09:26:00') < timestamp 
+and timestamp < toDateTime('2022-10-01 14:34:00')
+```
+
+> **Note:** by default, PostHog uses `UTC` for our timestamps, so you should keep that in mind and convert times to `UTC` where relevant.
+
+## Using the Unix timestamp
+
+If you have a Unix timestamp to filter by, you can use `toUnixTimestamp` to convert a date time object into a Unix timestamp, or `fromUnixTimestamp` to do the reverse. 
+
+For example, if you wanted to filter for events after the Unix timestamp `1674259200`, you could use the expression:
+
+```
+fromUnixTimestamp(1674259200) < timestamp
+```
+
+## Older or younger events
+
+If you are looking for events of a certain age, rather than a relation to a certain date, you can use the `dateDiff()` function with the timestamp and current time. 
+
+For example, to get events over 5 days old, use the expression:
+
+```
+dateDiff('day', timestamp, now()) > 5
+```
+
+You can replace `day` here with `second`, `minute`, `hour`, `week`, `month`, `quarter`, and `year` to get the differences in those values as well.
+
+## Weekly and quarterly reports
+
+Many companies report on a weekly or quarterly basis. For this, standard 7 or 90-day date range filters may include values beyond the days or weeks you want to report. HogQL lets us easily create these reports.
```

Comment:
> For this, standard 7 or 90-day date range filters may include values beyond the days or weeks you want to report.

This sentence is not clear. I'm not sure what it means.

Reply:
Rewrote